modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1244,6 +1255,26 @@ else if (clazz == Boolean.class) {
 
 			values.put(name, (Boolean)object);
 		}
+		else if (clazz == Clob.class) {
+			Clob clob = (Clob)object;
+
+			InputStream inputStream = null;
+			String value = null;
+
+			try {
+				inputStream = clob.getAsciiStream();
+
+				value = IOUtils.toString(inputStream, StandardCharsets.UTF_8);
+			}
+			catch (IOException ioException) {
```

Comment:
can we make like:

`catch (IOException | SQLException exception) {
throw new SystemException(exception);
}
`